lib/lita/handlers/versioner.rb

```diff
@@ -110,13 +110,14 @@ def github_handler(request, response)
           # https://developer.github.com/v3/activity/events/types/#pullrequestevent
           # If the pull request is merged with some commits
           if payload["action"] == "closed" && payload["pull_request"]["merged"]
+            inform("'#{payload["pull_request"]["html_url"]}' is just merged. Working on it...")
             new_version = bump_version_in_git
             trigger_build("v#{new_version}")
           else
-            inform("Looks like '#{payload["pull_request"]["url"]}' is not merged with commits. Skipping.")
+            log("Skipping: '#{payload["pull_request"]["html_url"]}' Action: '#{payload["action"]}' Merged? '#{payload["pull_request"]["merged"]}'")
```

Comment:
I think there is still a case we want to know about. What about this?

```
if payload["action"] == "closed"
  if payload["pull_request"]["merged"]
    ...
  else
    inform("Looks like PR X was closed without merging any commits. Skipping ...")
  end
else
  log("Skipping request because it's not closed (this is normal).")
end